ng-dev/utils/console.ts

```diff
@@ -103,7 +105,11 @@ function buildLogLevelFunction(loadCommand: () => Function, level: LOG_LEVELS) {
  */
 function runConsoleCommand(loadCommand: () => Function, logLevel: LOG_LEVELS, ...text: unknown[]) {
   if (getLogLevel() >= logLevel) {
-    loadCommand()(...text);
+    if (Spinner.activeSpinner) {
+      Spinner.activeSpinner.info(text.map((val) => inspect(val, true, 3, true)).join(' '));
+    } else {
+      loadCommand()(...text);
```

Comment:
I find it a little magical that any of the logging methods would interact with the spinner tbh. Can't we just print as always and reset the spinner to the lines after? e.g.

`/ Loading..` then we push a new message. We delete the current spinner line, call the original log method like before, and then re-start the spinner in a new line. Basically almost the same, but not doing anything with `inspect`, and without passing through the message to the spinner.